clearwater-cluster-manager.root/usr/share/clearwater/cluster_manager/scripts/load_from_cluster_settings.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import etcd
+import json
+
+local_ip = sys.argv[1]
+node_type = sys.argv[2]
+
+assert node_type in ["sprout", "ralf"], "Node type must be 'sprout' or 'ralf'"
+
+etcd_key = "/{}/clustering/memcached".format(node_type)
+
+def strip_port(server):
+    return server.rsplit(":", 1)[0].strip()
+
+with open('/etc/clearwater/cluster_settings') as f:
```

Comment:
This is technically configurable so I don't think we should just hard code it here